src/shared/ui/AButton/AButton.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import type { Meta, StoryFn } from '@storybook/vue3'
+import { AButton } from '.'
+
+const meta: Meta = {
+  title: 'UI/Button',
+  component: AButton,
+}
+
+export default meta
+
+export const Button: StoryFn = (props) => (
+  <AButton {...props}>
+    {{
```

Comment:
If you don't pass multiple slots, remove {{}} and pass content directly into component